src/ui/src/components/workflows/base/WorkflowArrow.vue

```diff
@@ -0,0 +1,80 @@
+<template>
+	<g @click="handleClick">
```

Comment:
You need to add `tabindex` to make it selectable using the keyboard
```suggestion
	<g tabindex="0" @click="handleClick">
```